pkg/debug/Dockerfile

```diff
@@ -70,6 +73,10 @@ RUN cp /etc/passwd /etc/group ./etc/
 RUN ln -s /run ./var/run
 RUN sed -i -e 's#AllowTcpForwarding.*$#AllowTcpForwarding yes#' ./etc/ssh/sshd_config
 
+# add recover-tpm to debug container
+WORKDIR /
+COPY --from=recovertpm /usr/bin/recovertpm /out/usr/bin/recovertpm
```

Comment:
Either the from isn't needed here, or hadlint is confused by the XXXX at the top of the file.

Reply:
fixed.